lib/pushy_client/protocol_handler.rb

```diff
@@ -489,7 +489,9 @@ def self.send_signed_message(socket, method, client_private_key, session_key, me
           socket.send_string(message)
         end
       rescue Timeout::Error
-        Chef::Log.info("ZMQ socket timed out")
+        Chef::Log.info("[#{node_name}] ZMQ socket timed out. Triggering reconfigure")
```

Comment:
I initially had the node_name in here, but we don't have access to the node_name method since this is a class method. 

```
ERROR: [pj-testing] Error in heartbeat / offline detection thread: undefined local variable or method `node_name' for PushyClient::ProtocolHandler:Class
/opt/push-jobs-client/embedded/lib/ruby/gems/2.4.0/gems/opscode-pushy-client-2.4.1/lib/pushy_client/protocol_handler.rb:492:in `rescue in send_signed_message'
```